_styles/docs.css

```diff
@@ -200,7 +200,9 @@ nav {
     color: #403757;
 }
 
-.sidebar > ul {
+.sidebar > ul,
```

Comment:
That would break any levels past the first three, and that matches the first one fine like that.